### ISSUE TITLE
CommandNotFoundError: update-rc.d during service.enabled on Centos 7

### ISSUE THREAD

**Issue by sknutsonsf** [saltstack/salt#32169]
### Description of Issue/Question
I'm installing services on Centos 7.2 and get the exception below. It happens with several different services.  I tried the "develop" branch yesterday and the issue is still occurring.
```py
==> default:      Comment: An exception occurred in this state: Traceback (most recent call last):
==> default:                 File "/usr/lib/python2.7/site-packages/salt/state.py", line 1703, in call
==> default:                   **cdata['kwargs'])
==> default:                 File "/usr/lib/python2.7/site-packages/salt/loader.py", line 1612, in wrapper
==> default:                   return f(*args, **kwargs)
==> default:                 File "/usr/lib/python2.7/site-packages/salt/states/service.py", line 469, in enabled
==> default:                   ret.update(_enable(name, None, **kwargs))
==> default:                 File "/usr/lib/python2.7/site-packages/salt/states/service.py", line 127, in _enable
==> default:                   if __salt__['service.enable'](name, **kwargs):
==> default:                 File "/usr/lib/python2.7/site-packages/salt/modules/systemd.py", line 704, in enable
==> default:                   cmd = [_get_service_exec(), '-f', name, 'defaults', '99']
==> default:                 File "/usr/lib/python2.7/site-packages/salt/modules/systemd.py", line 144, in _get_service_exec
==> default:                   salt.utils.check_or_die(executable)
==> default:                 File "/usr/lib/python2.7/site-packages/salt/utils/__init__.py", line 804, in check_or_die
==> default:                   raise CommandNotFoundError(command)
==> default:               CommandNotFoundError: update-rc.d
```
### Setup
(Please provide relevant configs and/or SLS files (Be sure to remove sensitive info).)

I've attached the config file for rabbitmq.  It has some other items related to rabbitmq and selinux

### Steps to Reproduce Issue
(Include debug logs if possible and relevant.)
[salt-debug-log.txt](https://github.com/saltstack/salt/files/191088/salt-debug-log.txt)
```
sudo sh install_salt.sh -P -X
unzip the files in  /vagrant or similar directory
ln -s /vagrant/salt-enable-bug /srv/salt
sudo salt-call state.highstate --local -l debug
```
Debug log output attached
[salt-enable-bug.zip](https://github.com/saltstack/salt/files/191087/salt-enable-bug.zip)
[salt-debug-log.txt](https://github.com/saltstack/salt/files/191089/salt-debug-log.txt)

### Versions Report
(Provided by running `salt --versions-report`. Please also mention any differences in master/minion versions.)
```yaml
Salt Version:
           Salt: 2015.8.8

Dependency Versions:
         Jinja2: 2.7.2
       M2Crypto: 0.21.1
           Mako: Not Installed
         PyYAML: 3.11
          PyZMQ: 14.7.0
         Python: 2.7.5 (default, Nov 20 2015, 02:00:19)
           RAET: Not Installed
        Tornado: 4.2.1
            ZMQ: 4.0.5
           cffi: Not Installed
       cherrypy: Not Installed
       dateutil: Not Installed
          gitdb: Not Installed
      gitpython: Not Installed
          ioflo: Not Installed
        libgit2: Not Installed
        libnacl: Not Installed
   msgpack-pure: Not Installed
 msgpack-python: 0.4.7
   mysql-python: Not Installed
      pycparser: Not Installed
       pycrypto: 2.6.1
         pygit2: Not Installed
   python-gnupg: Not Installed
          smmap: Not Installed
        timelib: Not Installed

System Versions:
           dist: centos 7.2.1511 Core
        machine: x86_64
        release: 3.10.0-327.el7.x86_64
         system: CentOS Linux 7.2.1511 Core
```

**Comment by tsaridas** [comment]
same error here.
```
salt-call --versions-report
Salt Version:
           Salt: 2015.8.8

Dependency Versions:
         Jinja2: 2.7.2
       M2Crypto: Not Installed
           Mako: Not Installed
         PyYAML: 3.11
          PyZMQ: 14.7.0
         Python: 2.7.5 (default, Nov 20 2015, 02:00:19)
           RAET: Not Installed
        Tornado: 4.2.1
            ZMQ: 4.0.5
           cffi: Not Installed
       cherrypy: Not Installed
       dateutil: Not Installed
          gitdb: Not Installed
      gitpython: Not Installed
          ioflo: Not Installed
        libgit2: Not Installed
        libnacl: Not Installed
   msgpack-pure: Not Installed
 msgpack-python: 0.4.6
   mysql-python: 1.2.3
      pycparser: Not Installed
       pycrypto: 2.6.1
         pygit2: Not Installed
   python-gnupg: Not Installed
          smmap: Not Installed
        timelib: Not Installed

System Versions:
           dist: centos 7.2.1511 Core
        machine: x86_64
        release: 3.10.0-327.10.1.el7.x86_64
         system: CentOS Linux 7.2.1511 Core
```

**Comment by jfindlay** [comment]
@sknutsonsf, @tsaridas, thanks for reporting.  It seems that `update-rc.d` is a debian-specific util that needs to be gated to debian platforms in the module.

**Comment by junster1** [comment]
We are impacted by this also on RHEL7.X.  Any eta on a fix?

**Comment by terminalmage** [comment]
Fixed in https://github.com/saltstack/salt/pull/32230

**Comment by junster1** [comment]
thanks @terminalmage this patch fixes the issue for us.  However I wonder if it might be safer in the long term to just call systemctl enable/disable directly and let it handle the underlying calls.  I know that works on RHEL, not sure on other flavors.

**Comment by themalkolm** [comment]
This bug doesn't exist in `2015.8.7`. So rolling back to this version works.

**Comment by terminalmage** [comment]
@junster1 I'll take a look at doing that, I'll need to make sure that it works on all releases that might use systemd.py.

**Comment by Northwood128** [comment]
Hi there! I have the same issue. Running Salt 2015.8.8.2

**Comment by Northwood128** [comment]
Fixed. I just installed python-systemd via pip.

**Comment by terminalmage** [comment]
@Northwood128 as noted [here](https://github.com/saltstack/salt/issues/32169#issuecomment-203210239), this has been fixed.

**Comment by emcgough** [comment]
But what version is it fixed in?   Still broken with Salt 2015.8.8-2.el7 on CentOS Linux release 7.2.1511 

The pip install python-systemd work around noted above did not work for me either.
